### PR TITLE
Fix/tao 6831 roles filtering

### PR DESCRIPTION
**Pull request by rkarolis**

### CODE REVIEW

File: views/js/controller/TaoEventLog/show.js [hunk 1]
```diff
@@ -152,7 +152,25 @@ define([
                     sortable: true,
                     filterable: true,
                     transform: function (roles) {
-                        return roles.split(', ').shift();
+                        var rolesArray = roles.split(', ');
+                        var rolesCount = rolesArray.length;
+
+                        if(rolesCount > 1) {
+                            if(currentFilter.filtercolumns.user_roles) {
+                                var roleFiltered = _.find(rolesArray, function (item) {
```
Comment (hutnikau) [inline views/js/controller/TaoEventLog/show.js]: Please define variables at the top of function.

File: views/js/controller/TaoEventLog/show.js [hunk 2]
```diff
@@ -152,7 +152,25 @@ define([
                     sortable: true,
                     filterable: true,
                     transform: function (roles) {
```
Comment (hutnikau) [inline views/js/controller/TaoEventLog/show.js]: Please use only one `return` statement. Multiple returns degrades readability.

**Comment by hutnikau** [comment]
In general looks good! Some minor changes required.
P.S. Please do the code formatting formatting in separate PR. It will make review much easier.

**Comment by rkarolis** [comment]
What do you mean about separate PR?

**Pull request merged**